src/content/app/tools/blast/components/blast-settings/BlastSettings.scss

```diff
@@ -35,17 +35,17 @@
   text-align: right;
 }
 
-.parametersColumn .matrixSetting .select label {
-  grid-template-columns: 140px 110px;
-}
-
 .blastSubmissionName label span,
 .select label span {
   font-weight: $light;
   line-height: 27px;
   justify-self: end;
 }
 
+.parametersColumn .matrixSetting .select label {
+  grid-template-columns: 140px 110px;
+}
```

Comment:
Nothing seems to have changed about this CSS block? Did you move it because stylelint was complaining about styles order, or because you wanted to change its CSS rules, but then decided not to? If it's the latter, could you please revert this change?

Reply:
Yes because of the linter complaining about its order.